lib/index.js

```diff
@@ -215,13 +203,71 @@ function lowerCaseObjectKeys(obj) {
   return newObject;
 }
 
+function validateDate(date, key) {
+  // created & expires are not required
+  if(date === undefined || date === null) {
+    return date;
+  }
+  if(date instanceof Date) {
+    const timestamp = date.getTime() / 1000;
+    if(key === 'created') {
+      return Math.ceil(timestamp);
+    }
+    if(key === 'expires') {
+      return Math.floor(timestamp);
+    }
```

Comment:
In thinking about this more, if people are passing in a Date object, chances are they would be more surprised if `created` rounded up than down. The case to be concerned about with rounding down for `created` is really around people passing in milliseconds, but that's not even an option, it seems. If someone started with milliseconds and parsed it into a Date, then that will be their problem. Let's simplify:

```suggestion
    const timestamp = date.getTime() / 1000;
    if(key === 'created' || key === 'expires') {
      return Math.floor(timestamp);
    }
```